### DISCUSSION TITLE
Alternative to GitHub Achievements

### DISCUSSION THREAD

**Discussion by annamedv** [Schweinepriester/github-profile-achievements#849] [Ideas]
Hey 👋🏻

While GitHub Achievements are great, they lack community contributions. It will be so cool if used can define own badges. 

This is exactly what [my-badges](https://github.com/my-badges/my-badges) project is doing. It is a community driven achievements for profile. 

Check it out!

* https://github.com/my-badges/my-badges

**Comment by Schweinepriester** [comment]
Cool project! 

I'm gonna move this to discussions though 0:-)
